lib/librte_hash/rte_hash_version.map

```diff
@@ -45,3 +44,12 @@ DPDK_16.07 {
 	rte_hash_get_key_with_position;
 
 } DPDK_2.2;
+
+EXPERIMENTAL {
+	global:
+
+	rte_hash_iterator_init;
+	rte_hash_iterate;
```

Comment:
It's my understanding that `rte_hash_iterator_init()` and `rte_hash_iterate` should not be experimental since they are only a rewrite of an essential function of the library. The other functions are being marked experimental mostly because they are adding functionality to the library and the developers want to play safe before marking them as official part of the library.